docker/entrypoint.sh

```diff
@@ -3,9 +3,11 @@
 RADICALE_UID="${RADICALE_UID:-1000}"
 RADICALE_GID="${RADICALE_GID:-1000}"
 
-addgroup   -g ${RADICALE_GID} radicale
-adduser -D -s /bin/false -u ${RADICALE_GID} -G radicale radicale
+getent group  radicale &> /dev/null || addgroup -g ${RADICALE_GID} radicale
+getent passwd radicale &> /dev/null || adduser -D -H -s /bin/false -u ${RADICALE_GID} -G radicale radicale
 
-chown -R radicale:radicale "$(dirname "${RADICALE_CONFIG:-/etc/radicale/radicale.config}")" /var/lib/radicale
+RADICALE_CONFIG="${RADICALE_CONFIG:-/etc/radicale/radicale.config}"
 
-exec su - radicale -s /bin/sh -c "exec radicale --hosts 0.0.0.0:5232 ${RADICALE_CONFIG:+--config='${RADICALE_CONFIG}'} $*"
+chown -R radicale:radicale $(dirname $RADICALE_CONFIG) /var/lib/radicale
+
+exec su -s /bin/sh -c "exec radicale --hosts 0.0.0.0:5232 $*" radicale
```

Comment:
`RADICALE_CONFIG` env variable won't be visiable to the radicale process anymore. That's the reason I chose to pass it as argument `${RADICALE_CONFIG:+--config='${RADICALE_CONFIG}'}`

Reply:
right, but i remove setting the default then as it sets an uncommon location. `/etc/radicale/config` is found anyway.